pages/linux/systemd-tmpfiles.md

```diff
@@ -0,0 +1,24 @@
+# systemd-tmpfiles
+
+> Create, delete and clean up volatile and temporary files and directories.
+> More information: <https://www.freedesktop.org/software/systemd/man/systemd-tmpfiles.html>.
+
+- Create files and directories as specified in the configuration:
+
+`systemd-tmpfiles --create`
+
+- Clean up files and directories with age parameters configured:
+
+`systemd-tmpfiles --clean`
+
+- Remove files and directories as specified in the configuration:
+
+`systemd-tmpfiles --remove`
+
+- Apply operations for user-specific configurations::
```

Comment:
```suggestion
- Apply operations for user-specific configurations:
```

Reply:
I think we should add `::` detection to tldr-lint, this issue seems to be occurring pretty often and sometimes goes through our oversight too.

Reply:
@kbdharun I can work on it if there is an issue created